packages/quantic/cypress/e2e/facets-1/facet/facet-with-captions.cypress.ts

```diff
@@ -0,0 +1,62 @@
+import {performSearch} from '../../../page-objects/actions/action-perform-search';
+import {configure} from '../../../page-objects/configurator';
+import {getQueryAlias, interceptSearch} from '../../../page-objects/search';
+import {
+  useCaseEnum,
+  InsightInterfaceExpectations,
+  useCaseParamTest,
+} from '../../../page-objects/use-case';
+import {BreadcrumbManagerExpectations} from '../../facets-2/breadcrumb-manager/breadcrumb-manager-expectations';
+import {FacetActions as Actions} from './facet-actions';
+import {FacetExpectations as Expect} from './facet-expectations';
+
+interface FacetCaptionOptions {
+  useCase: string;
+  value: string;
+  caption: string;
+}
+
+describe('Facet with captions Test Suite', () => {
```

Comment:
Just for consistency with category facet test suit:
```suggestion
describe('quantic-facet with captions', () => {
```